### DISCUSSION TITLE
config.ini "file does not exist"

### DISCUSSION THREAD

**Discussion by okayyeahsure** [tgorgdotcom/locast2plex#157] [Q&A] [answered]
I had locast2plex working and up in Docker for quite a while, and all of the sudden I am getting an error saying that the config.ini does not exist. The file is still there and def does exist. I am not sure what is happened. 

Ultimately I deleted my Docker config and started fresh, making a new config.ini file and put it in a different folder. When I try to install again, I get this error in terminal...

docker run -v /Users/REDACTED/Desktop/config.ini:/app/config/config.ini -p 6077:6077 -p 1900:1900/udp tgorg/locast2plex 
docker: Error response from daemon: Mounts denied: approving /Users/REDACTED/Desktop/config.ini: file does not exist.
ERRO[0000] error waiting for container: context canceled

When I try to run in Docker, I get this error:
(HTTP code 500) server error - Mounts denied: approving /Users/REDACTED/Desktop/config.ini: file does not exist

Like I said, everything was running fine for months and then it just stopped working. Anybody have any idea what might be happening?

**Comment by arejay73** [comment, marked as answer]
Docker flipped a bit with the 3.0 upgrade.  See this [stack overflow](https://stackoverflow.com/questions/65245445/docker-error-response-from-daemon-mounts-denied-approving-path-to-file-does) post.

**Reply by okayyeahsure** [reply]
It works! Thank you!
